packages/camera/camera_web/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2013 The Flutter Authors. All rights reserved.
+Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
This change (in all files, not just here) needs to be reverted. It's why `format` is red.

Reply:
Reverted the change, but `format` keeps failing. 🤔  Do you have any idea why that might be?